Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -367,10 +360,17 @@ public enum TextfieldType: String {
             self.setURLStackView(hiddenStatus: true)
         }
         
+        //set username and password fields status (enabled/disabled)
+        let enabledEditUrlUsernamePassword : Bool = (self.loginMode == .create || self.loginMode == .migrate)
+        self.textFieldURL.isEnabled = enabledEditUrlUsernamePassword
+        self.textFieldURL.isUserInteractionEnabled = enabledEditUrlUsernamePassword
+        self.textFieldUsername.isEnabled = true
+        self.textFieldUsername.isUserInteractionEnabled = true
+        
         self.textFieldUsername.autocorrectionType = .no
         self.textFieldURL.autocorrectionType = .no
         
-        //set user&pass visibility
+        //set username and password fields visibility
```

Comment:
I think that if you need to write down this comment maybe you should change the name of the method in order to explain better what it does.